experiment_buddy/experiment_buddy.py

```diff
@@ -219,7 +219,7 @@ def _ask_experiment_id(cluster, sweep):
         experiment_id = tkinter.simpledialog.askstring(title, "experiment_id")
         root.destroy()
     except:
-        experiment_id = input(f"Running on {title} \ndescribe your experiment (experiment_id):\n")
+        experiment_id = input(f"Running on {title} 😀\ndescribe your experiment (experiment_id):\n")
```

Comment:
nofunallowed.jpg
i'm afraid it will break some shells :(

Reply:
It doesn't, but this was committed by mistake, reverting